### PR TITLE
Add asciidoc support and import intro-vm-portal

### PR DESCRIPTION
**Pull request by pmkovar**
Fixes issue #2086

Changes proposed in this pull request:

* Add support for font-based admonitions & coderay highlighter

  Based on 0a135ac2f93505274d133aa797371b1e9fae0c4e and a successful jekyll-asciidoc deployment at:
https://github.com/eclipse/che-docs/

* Import asciidoc version of intro-vm-portal guide for ovirt 4.3 as a proof of concept

  Based on the common attributes setup by @rolfedh.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @pmkovar

This pull request needs review by: @sandrobonazzola @duck-rh

### CODE REVIEW

**Comment by pmkovar** [comment]
Can confirm that the `listen` gem keeps complaining about symlinks (https://github.com/guard/listen/wiki/Duplicate-directory-errors) at build time. Can we deploy anyway?

Tried to workaround that problem with 484d0cdef99605378f7cf5330bf8b27428f8442d, but interestingly enough that triggers another problem, probably related to the fact `asciidoctor` won't allow includes from the index file's parent directory for security reasons even in unsafe mode:

```
ERROR Errno::ECONNRESET: Connection reset by peer @ io_fillbuf - fd:16 
	/usr/share/ruby/webrick/httpserver.rb:82:in `eof?'
	/usr/share/ruby/webrick/httpserver.rb:82:in `run'
	/usr/share/ruby/webrick/server.rb:307:in `block in start_thread'
```
We could try placing a master index file under ` source/documentation/` and include all guides from that file, but we don't want to deviate too much from the downstream setup because of content sharing. 

Other ideas?

**Comment by sandrobonazzola** [comment]
@duck-rh can you help answering @pmkovar questions?

**Comment by duck-rh** [comment]
@sandrobonazzola when I come back from event and vacation on 18th I will, sorry for the delay

**Comment by sandrobonazzola** [comment]
Hey @duck-rh any update?

File: _config.yml [hunk 1]
```diff
@@ -44,6 +44,23 @@ kramdown:
   parse_block_html: true
   parse_span_html: true
 
+plugins:
+  - jekyll-asciidoc
+
+asciidoc: {}
```
Comment (duck-rh) [inline _config.yml]: This is not needed / legacy, the `asciidoctor` entry is sufficient.

**Comment by pmkovar** [comment]
> If you move `source/documentation/common` out of the source directory (like a `adoc-common` top directory or the like and update the symlink accordingly), then it fixes your problem. Also Jekyll will generate any subpage in common, so it's better to move it aside to avoid broken generation and having unwanted pages.

@duck-rh  Done, thanks for the suggestion.
 
> In fact I would suggest finding a way to move all includes away and keep only the index.adoc unless we want split pages.

Understood, but I want to keep the changes to the asciidoc sources to a bare minimum.

**Comment by sandrobonazzola** [comment]
@duck-rh can you please review again?

**Comment by duck-rh** [comment]
@sandrobonazzola  Same remarks as #2177 apply.

Both PRs embed the asciidoc activation and coderay addition, so one should go in first and the other be rebase on top of it.

So once it is done in either of them I can do a final review.

**Comment by pmkovar** [comment]
> @sandrobonazzola Same remarks as #2177 apply.
> 
> Both PRs embed the asciidoc activation and coderay addition, so one should go in first and the other be rebase on top of it.

@sandrobonazzola @duck-rh Let's merge this one first. #2177 is a follow up.

**Pull request merged**